### ISSUE TITLE
[autofix] Using --fix on single line comment on else line remove space

### ISSUE THREAD

**Issue by Cactusbone** [stylus/stylelint-stylus#18]
when using `--fix`, spaces between `else` and comments are removed

```
    if foo // some comment
        width 100%
    else // another
        width 33%
```
becomes
```
    if foo // some comment
        width 100%
    else// another 
        width 33%
```

this does not break anything, but this behaviour was not expected.

It should also be noted that a space was added at EOL, thus breaking stylus/no-eol-whitespace rule on next lint

**Comment by ota-meshi** [comment]
Fixed in [postcss-styl v0.12.3](https://github.com/stylus/postcss-styl/releases/tag/v0.12.3). Please update postcss-style.

**Comment by Cactusbone** [comment]
I'm not sure what happened here ... the bug report was for `stylelint-stylus`, not `postcss-styl`
